LabBuilder/LabBuilder.psm1

```diff
@@ -2774,7 +2889,8 @@ Add-Content ``
     $SetupCompleteCmd = @"
 @echo SetupComplete.cmd Script Started... >> %SYSTEMROOT%\Setup\Scripts\SetupComplete.log
 $SetupCompleteCmd
-powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`"
+Timeout 30
+powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`" `r
```

Comment:
Cool - lets use your fix. And if I run into any problems using it on my labs we can figure something else out.